### DISCUSSION TITLE
Upptime-bot is deleting gh-pages

### DISCUSSION THREAD

**Discussion by cixastra** [upptime/upptime#401] [Q&A] [answered]
Note before I start: to avoid the "build hours" I have used [Vercel](https://vercel.com) to deploy the website. 
- My Repo is Private

When setup Upptime completes, @upptime-bot makes a commit to delete all of my static site files except 404. :man_facepalming: I know it has something to do with the fact that I am not using GH pages... however, I don't know how to fix this without using GH Pages. I want to continue with [Vercel](https://vercel.com) if I can, as I am familiar with it..

**Comment by cixastra** [comment, marked as answer]
Disregard this :)

**Reply by foxted** [reply]
How did you fix this? I'm having the same issue :/

**Reply by cixastra** [reply]
@foxted I wasn't able to solve it, I simply archived the project. I also assume that since the project is so deeply geared towards github's internal functions and their own connections that using something like vercel wouldn't be possible. But this is conjecture, feel free to open another discussion and reference this, or I can unresolve this one. :)

**Reply by redlinejoes** [reply]
You should leave issues unresolved unless they are resolved.

**Comment by MatadorProBr** [comment]
This is because you make your repo private, so it's not possible to have a status page with the data of a private repository, the only solutions for this is: make the repo public, https://upptime.js.org/docs/faq#my-status-website-doesnt-work-with-my-private-repository or https://github.com/upptime/upptime/discussions/54

**Reply by foxted** [reply]
I ended up building a small proxy to solve that issue.

**Reply by redlinejoes** [reply]
This is happening to me as well. I am using GitHub Pages, and the upptime bot continues to delete my gh-pages files. 

This should be marked as unresolved unless a solution is provided. 

Please explain in more detail, "I ended up building a small proxy to solve that issue."

My repo is currently public. 

None of these suggestions apply to the issue where the upptime bot deletes the content of a public repo gh-pages.
